config/routes.rb

```diff
@@ -26,5 +26,6 @@
 
   namespace :admin do
     resources :trips, only: %i[edit update destroy]
+    resources :conditions, only: %i[index edit update show]
```

Comment:
I put the button logic in the normal view instead of having new views. How does everyone feel about this?

Reply:
let's have a conversation about this and do it as a refactor.